src/utils/control-icon-manager.ts

```diff
@@ -0,0 +1,46 @@
+import { FullScreen, Zoom } from 'ol/control';
+import { useStore } from '../composable/store';
+import SVGCreator from './svg-creator';
+import GeolocationCenter from '../components/geolocation-center';
+import ResetRotationControl from '../components/reset-rotation-control';
+import InformationControl from '../components/information-control';
+
+export default class ControlIconManager {
```

Comment:
actually would probably simplify a lot the css if you would put all control icon inside a container control.
This way you can define the display of buttons inside the container with flex if needed and only change the margin-top of the buttonsContainer when info box is displayed
Something like:
```
class ButtonsContainer extends Control {
  /**
   */
public div: HTMLElement;

  constructor() {
    const element = document.createElement('div');
    element.className = 'buttons-control-container';

    super({
      element: element,
    });

    this.div = element;
  }

}

export default class ControlIconManager {
  static setupIcon() {
    const options = useStore().getOptions();
    const map = useStore().getMap()
    map.addControl(new InformationControl());

    const buttonsContainer = new ButtonsContainer();
    map.addControl(buttonsContainer);
    if (options.fullscreen)
      map.addControl(
        new FullScreen({
          label: SVGCreator.fullScreenLabel(),
          labelActive: SVGCreator.fullScreenLabelActive(),
          className: `ol-full-screen`,
          target: buttonsContainer.div
        })
      );
```

Reply:
Done but with a little change. I have two container. One for the left side and one for the right side.